docs/sp-migration-mr.md

```diff
@@ -58,22 +58,22 @@ spec:
 done
 ```
 
-4. Install config providers with `revisionActivationPolicy: Manual`:
+4. Install family providers with `revisionActivationPolicy: Manual`:
```

Comment:
Having duplicated steps across two guides is bugging me. 
@ezgidemirel do you see a way to combine them?

Reply:
we can mark some of the steps as `configuration only` but I think it will make things more complicated for the user.